openfermioncirq/trotter/algorithms/linear_swap_network.py

```diff
@@ -115,16 +115,19 @@ def trotter_step(
 
         n_qubits = len(qubits)
 
+        if not isinstance(control_qubit, cirq.Qid):
+            raise TypeError('Control qudit must be specified.')
+
         # Apply one- and two-body interactions for half of the full time
         def one_and_two_body_interaction(p, q, a, b) -> cirq.OP_TREE:
             yield CRxxyy(
                     0.5 * self.hamiltonian.one_body[p, q].real * time).on(
-                            control_qubit, a, b)
+                            cast(cirq. Qid, control_qubit), a, b)
```

Comment:
`cirq. Qid` -> `cirq.Qid`  [or is there something wrong with my font?]

Reply:
Done. How did that not get picked up by a test?!

Reply:
Apparently it still works with the space there. But perhaps pylint should have complained